core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -236,6 +237,7 @@ object KafkaConfig {
   val NumIoThreadsProp = "num.io.threads"
   val BackgroundThreadsProp = "background.threads"
   val QueuedMaxRequestsProp = "queued.max.requests"
+  val QueuedMaxBytesProp = "queued.max.bytes"
```

Comment:
Would it be better to name this queued.max.request.bytes? Otherwise, it's not obvious what queued bytes are for.